mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -61,6 +61,10 @@
     'custom': [],
 }  # type: T.Dict[str, T.List[str]]
 
+# Clang-cl doesn't have /ZI, and /Zi and /Z7 do the same thing
```

Comment:
Perhaps this could be clearer about what the "same thing" that these are doing is?  (Not quite clear what exactly that is, though? Producing CV debug info, which the linker is expected to separate to a PDB?)